src/node_config.cc

```diff
@@ -42,9 +42,7 @@ static void Initialize(Local<Object> target,
   READONLY_FALSE_PROPERTY(target, "hasOpenSSL");
 #endif  // HAVE_OPENSSL
 
-#ifdef NODE_FIPS_MODE
   READONLY_TRUE_PROPERTY(target, "fipsMode");
```

Comment:
Should this reflect `FIPS_mode()` instead of always being set to true?

Reply:
That is very good point looking into this. Do you think it would be OK to basically remove the `fipsMode` flag and its usage in crypto.js?

Reply:
Since this is internal it's probably okay to remove, but you'll need to update everywhere that refers to it, i.e. any of the files under `lib` or `test` that do:
```js
const { fipsMode } = internalBinding('config');
```